symphony/bdk/core/service/message_service.py

```diff
@@ -0,0 +1,366 @@
+            params_map={
+                'all': [
+                    'sid',
+                    'session_token',
+                    'key_manager_token',
+                    'message',
+                    'data',
+                    'version',
+                    'attachment',
+                    'preview',
+                ],
+                'required': [
+                    'sid',
+                    'session_token',
+                ],
+                'nullable': [
+                ],
+                'enum': [
+                ],
+                'validation': [
+                ]
+            },
+            root_map={
+                'validations': {
+                },
+                'allowed_values': {
+                },
+                'openapi_types': {
+                    'sid':
+                        (str,),
+                    'session_token':
+                        (str,),
+                    'key_manager_token':
+                        (str,),
+                    'message':
+                        (str,),
+                    'data':
+                        (str,),
+                    'version':
+                        (str,),
+                    'attachment':
+                        ([file_type],),
+                    'preview':
+                        ([file_type],),
+                },
+                'attribute_map': {
+                    'sid': 'sid',
+                    'session_token': 'sessionToken',
+                    'key_manager_token': 'keyManagerToken',
+                    'message': 'message',
+                    'data': 'data',
+                    'version': 'version',
+                    'attachment': 'attachment',
+                    'preview': 'preview',
+                },
+                'location_map': {
+                    'sid': 'path',
+                    'session_token': 'header',
+                    'key_manager_token': 'header',
+                    'message': 'form',
+                    'data': 'form',
+                    'version': 'form',
+                    'attachment': 'form',
+                    'preview': 'form',
+                },
+                'collection_format_map': {
+                }
+            },
+            headers_map={
+                'accept': [
+                    'application/json'
+                ],
+                'content_type': [
+                    'multipart/form-data'
+                ]
+            },
+            api_client=api_client,
+            callable=__v4_message_create_post
+        )
+
+        self.v4_multi_attachment_message_blast_post = Endpoint(
+            settings={
+                'response_type': (V4MessageBlastResponse,),
+                'auth': [],
+                'endpoint_path': '/v4/message/blast',
+                'operation_id': 'v4_message_blast_post',
+                'http_method': 'POST',
+                'servers': None,
+            },
+            params_map={
```

Comment:
looks like from here it is similar to above, maybe we can factorize it?

Reply:
actually we have a slight difference between `sid` and `sids`